parkings/api/operator/parking.py

```diff
@@ -57,6 +57,10 @@ def update(self, instance, validated_data):
         return instance
 
     def validate(self, data):
+        if self.instance and (now() - self.instance.created_at) > settings.PARKINGS_TIME_EDITABLE:
+            if set(data.keys()) != {'time_end'}:
+                raise exceptions.PermissionDenied(_('Other than PATCHing "time_end" not allowed anymore.'))
```

Comment:
Talked IRL that 403 was seen in previous discussions in similar API situations as the correct code to return in this kind of validation situation where "train has passed" even though this has absolute nothing to do with permissions.

Still, error message needs some language correctness 👍 Once the error message has been tweaked, ready to 🚢